src/tests/transfer_test.ts

```diff
@@ -114,7 +126,15 @@ describe("Transfer Class", () => {
     });
   });
 
-  describe("getTransaction", () => {
+  describe("#getTransactionLink", () => {
+    it("should return the transaction link when the transaction hash is available", () => {
+      expect(transfer.getTransactionLink()).toEqual(
+        `https://sepolia.basescan.org/tx/${transferModel.transaction_hash}`,
```

Comment:
Should we use the transaction link property on the transfer model?

https://github.com/coinbase/coinbase-sdk-nodejs/blob/aaaf99926847e316b8cf51c0699b2971fb1548b8/src/coinbase/transaction.ts#L101
https://github.com/coinbase/coinbase-sdk-ruby/blob/c836dedacd7c337b4f6cd85f00fd049562cad79a/lib/coinbase/transaction.rb#L78-L80

Can land in follow-up?

Reply:
Sure, I'm going to fix that in my next PR 👍